### ISSUE TITLE
Luazen

### ISSUE THREAD

**Issue by SWSAmor** [tarantool/tarantool#3245]
I'd like to see this library
https://github.com/philanc/luazen
integrated in Tarantool. 
Especially the Curve25519-based key exchange and the Ed25519 signature would be great. (LZF is good too.)
Thx.

**Comment by kostja** [comment]
Yes, it would be great. Do you want to encode/decode compress in tarantool thread pool?
Care to submit a rockspec pull request to https://github.com/tarantool/rocks?

**Comment by kostja** [comment]
Basically, all you need to do to integrate it is to fork it into our organization and add .travis.yml which would build it along with our docker, as well as in our rpm/deb reposytories.

**Comment by Totktonada** [comment]
Argon2i is interesting option to have it (more or less) on-board for strict security apps with password hashing. Don’t sure why not Argon2id was implemented, though.

**Comment by SWSAmor** [comment]
I'm scared. :-) I've never done this before.

**Comment by kyukhin** [comment]
@SWSAmor, should we wait for PR?

**Comment by SWSAmor** [comment]
No, please, i’m busy right now.
